FILE packages/desktop/electron-builder.yml
RANGE {"path":"packages/desktop/electron-builder.yml","old_lines":[1,2],"new_lines":[1,3],"diff_block_lines":[5,8],"hunk_header":"@@ -1,2 +1,3 @@","substantive_changed_lines":1}
@@ -1,2 +1,3 @@
+npmRebuild: false
 appId: sh.paseo.desktop
 productName: Paseo
